app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<p>Please provide the following information to <%= verb_name %> your task to our database:</p>
+
+<%= form_with model: @task, class: "task-input" do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+
+  <%= f.label :description %>
+  <%= f.text_area :description %>
+
+  <%= f.label :completion_date %>
+  <%= f.date_select :completion_date %>
+
+  <%= f.label :completed %>
+  <%= check_box 'task', 'completed', {}, 'true', 'false' %>
```

Comment:
You may have missed the request, but I believe we said that the optimal way to implement this was to use `completion_date` as the way you check to see if a task is complete or not. This way you'd be toggling between setting the completion_date to `Time.now` and `nil`.